### PR TITLE
Vote-1639 -> Updated Navajo String Translations 

### PR DESCRIPTION
**Pull request by clmedders**
**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1639](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1639&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This PR will update the Navajo sting translations to match the current live site. 

### Pre-deploy steps
1. Pull down branch 
2. run `lando import-translations`

### Testing steps
1. start local 
2. visit `https://vote.gov/nv/register/ak/`, `https://vote.gov/nv/register/as/`, `https://vote.gov/nv/register/ar/`, `https://vote.gov/nv/register/nd/`,  and `https://vote.gov/nv/register/wy/` and compare them with the local build (you will need to log in and can run `lando drush uli` to do so). 
3. local build URLs `http://vote-gov.lndo.site/nv/register/ak/`, `http://vote-gov.lndo.site/nv/register/as/`, `http://vote-gov.lndo.site/nv/register/ar/`, `http://vote-gov.lndo.site/nv/register/nd/`,  and `http://vote-gov.lndo.site/nv/register/wy/`

### CODE REVIEW

File: web/modules/custom/vote_utility/translations/nv.po [hunk 1]
```diff
@@ -13,214 +8,285 @@ msgstr ""
...
 
+# config.languages.params.select_language
 msgid "Select language"
 msgstr "Saad choid77['88[7g77"
 
-#main
+# config.languages.params.Owner
 msgid "@sitename in language"
-msgstr "@sitename Diné"
+msgstr "@sitename in Diné"
```
Comment (rayestrada) [inline web/modules/custom/vote_utility/translations/nv.po:21]: Remove `in`. Compare with the live site.

File: web/modules/custom/vote_utility/translations/nv.po [hunk 2]
```diff
@@ -13,214 +8,285 @@ msgstr ""
...
 msgid "Go back"
 msgstr "N1t'33'"
 
-msgid "Illustration of voter box with checked ballot being inserted to indicate voting action"
-msgstr "&'ii'n77[ b1 sits'aa y1zh7 si'1n7g77 biyi' asdzoh naashch'22'7g77 47 n7l47 o'oot'ah n7igo yaa y1[ti'go si'32go 1lyaa"
-
+# config.languages.params.last_updated
 msgid "Last updated:"
-msgstr "Ak44'di n1zk'33z:"
+msgstr "Ak44'di n1zk'33z: "
```
Comment (rayestrada) [inline web/modules/custom/vote_utility/translations/nv.po:29]: remove empty space at end of string.

File: web/modules/custom/vote_utility/translations/nv.po [hunk 3]
```diff
@@ -13,214 +8,285 @@ msgstr ""
...
+
+# config.languages.params.ext_link_title
+msgid "External link opens in new window"
+msgstr "External link opens new window"
```
Comment (rayestrada) [inline web/modules/custom/vote_utility/translations/nv.po:33]: `msgstr "External link opens in new window"`

File: web/modules/custom/vote_utility/translations/nv.po [hunk 4]
```diff
@@ -13,214 +8,285 @@ msgstr ""
...
+msgstr "Contact"
+
+# data.translations.footer.gsa_logo_alt
+msgid "GSA logo"
+msgstr "GSA logo"
+
+# data.translations.homepage.search_placeholder
 msgid "Search Vote.gov"
-msgstr "vote.gov hanit1h"
+msgstr "vote.gov"
```
Comment (rayestrada) [inline web/modules/custom/vote_utility/translations/nv.po:105]: this should be 
`msgstr "vote.gov hanit1h"`

File: web/modules/custom/vote_utility/translations/nv.md
Comment (rayestrada) [inline web/modules/custom/vote_utility/translations/nv.md:1]: Delete this md file.

File: web/modules/custom/vote_utility/translations/nv.po [hunk 5]
```diff
@@ -13,214 +8,285 @@ msgstr ""
...
+msgid "Illustration of voter box with checked ballot being inserted to indicate voting action"
+msgstr "&'ii'n77[ b1 sits'aa y1zh7 si'1n7g77 biyi' asdzoh naashch'22'7g77 47 n7l47 o'oot'ah n7igo yaa y1[ti'go si'32go 1lyaa "
+
+# data.translations.footer.identifier_gsa_txt
 msgid "An official website of the @link."
 msgstr "!l32j8' han1lki'7g77 website kod00 @link"
 
+# data.translations.footer.identifier_gsa_txt__1
 msgid "General Services Administration"
 msgstr "General Services Administration woly4ego t'11 a[tso yee 1k1 an7daalwo'd00 &lt;Bilag1ana k'ehj7&gt;ß"
```
Comment (rayestrada) [inline web/modules/custom/vote_utility/translations/nv.po:53]: I looked at the original translation file and this character at the end needs to be removed `ß﻿`

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: web/modules/custom/vote_utility/translations/nv.po (hunk 5 above)
Comment (clmedders) [reply web/modules/custom/vote_utility/translations/nv.po:53]: removed tailing character

File: web/modules/custom/vote_utility/translations/nv.po (hunk 2 above)
Comment (clmedders) [reply web/modules/custom/vote_utility/translations/nv.po:29]: removed tailing space at the end of string

File: web/modules/custom/vote_utility/translations/nv.po (hunk 3 above)
Comment (clmedders) [reply web/modules/custom/vote_utility/translations/nv.po:33]: fixed string

File: web/modules/custom/vote_utility/translations/nv.po (hunk 4 above)
Comment (clmedders) [reply web/modules/custom/vote_utility/translations/nv.po:105]: added missing translation.

File: web/modules/custom/vote_utility/translations/nv.po (hunk 1 above)
Comment (clmedders) [reply web/modules/custom/vote_utility/translations/nv.po:21]: removed 'in' to match live site

File: web/themes/custom/vote_gov/templates/block/block-content--email-signup--full.html.twig [hunk 6]
```diff
@@ -7,7 +7,7 @@
       {# what goes in action #}
         <form class="form-signup" action="{{ content.field_link | field_value }}" method="get">
           <h2 class="text-color-blue font-size-medium">
-              {{ content.field_heading | field_value | render | replace({'%USAGov%': 'USAGov'}) }}
+              {{ content.field_heading | field_value | render | replace({'%USAGov%': 'USAGov'}) | raw}}
```
Comment (clmedders) [inline web/themes/custom/vote_gov/templates/block/block-content--email-signup--full.html.twig:10]: updated the text in the dev but also saw that it was giving the same issue as the nvrf and added this and its working on my local as well

**Review by rayestrada — APPROVED** [review]

**Pull request merged**